packages/lesswrong/components/ea-forum/wrapped/EAForumWrapped2023Page.tsx

```diff
@@ -76,16 +73,12 @@ const styles = (theme: ThemeType) => ({
   '@keyframes section-scroll-animation': {
     '0%': {
       opacity: 0,
-      // TODO: this messes with scroll snapping, see if we can add it back in
-      // transform: 'scale(0.5)'
```

Comment:
I played around with this but I didn't like it without the scroll snapping, so I just gave up on it